skimage/filters/_gaussian.py

```diff
@@ -155,3 +155,132 @@ def _guess_spatial_dimensions(image):
         return 3
     else:
         raise ValueError("Expected 2D, 3D, or 4D array, got %iD." % image.ndim)
+
+
+def difference_of_gaussians(image, low_sigma, high_sigma=None, *,
+                            mode='nearest', cval=0, multichannel=False,
```

Comment:
Isn't it better to have `multichannel` as the last argument here?

Reply:
they are keyword-only arguments, so this doesn't matter at all, we can reorder them any time we like. That's the point! =D

Reply:
But... but we like order! ... don't we? :)

Reply:
I actually just followed the ordering in `gaussian`, except adding the `*` as @jni noted.

Reply:
Don't worry @mellertd, the order is different in my head :)